### PR TITLE
Updates specs to use RSpec 3 syntax.

### PR DESCRIPTION
**Pull request by afred**

### CODE REVIEW

**Comment by afred** [comment]
@mccalluc hooray for green. Take a peek please, and assure me of no egregious grossness. I'm going to rebase outstanding PRs to see if we can get continued green.

**Comment by afred** [comment]
^^ that is, I'll rebase outstanding PRs onto `wgbh-master` after you merge this one.

File: spec/models/generic_file_spec.rb [hunk 1]
```diff
@@ -65,14 +65,14 @@
 </ffprobe>
 EOF
       end
-      its(:video?) { should be_false}
-      its(:audio?) { should be_true}
+      its(:video?) { should eq false}
+      its(:audio?) { should eq true}
```
Comment (mccalluc) [inline spec/models/generic_file_spec.rb]: Are these `should`s ok? I'm not sure how they would be translated.

File: spec/unit/samfs_storage_manager_spec.rb [hunk 2]
```diff
@@ -40,7 +40,7 @@
 )
       end
       it "should be false" do
-        subject.live?('/opt/testfile').should be_false
+        subject.live?('/opt/testfile').should eq false
```
Comment (mccalluc) [inline spec/unit/samfs_storage_manager_spec.rb]: `expect`?

**Comment by mccalluc** [comment]
@afred: None of these are wgbh specific... Should they me made on the 1.x branch instead?

File: spec/unit/samfs_storage_manager_spec.rb (hunk 2 above)
Comment (afred) [inline spec/unit/samfs_storage_manager_spec.rb]: ah.. yes. Stand by...

File: spec/models/generic_file_spec.rb (hunk 1 above)
Comment (afred) [inline spec/models/generic_file_spec.rb]: These calls to `should` i believe are part of the `rspec-its` backport. If you want to convert these to use `expect` syntax, it's kind of a pain... which is why i included `rspec-its` in the first place.

**Pull request closed without merging**